app/controllers/coronavirus_form/check_answers_controller.rb

```diff
@@ -64,6 +74,14 @@ def user_email
     end
   end
 
+  def user_mobile_number
+    TelephoneNumber.parse(session_with_indifferent_access.dig(:contact_details, :phone_number_texts), :gb).national_number(formatted: false)
+  end
+
+  def mobile_number_provided?
+    TelephoneNumber.valid?(session_with_indifferent_access.dig(:contact_details, :phone_number_texts), :gb, [:mobile])
```

Comment:
We'll need to remove whitespace from the mobile number here, if this isn't done elsewhere.

Reply:
I think we will have done this when the number is stored in the contact details controller, since the number is being saved as `national_number` which will have already been formatted by the TelephoneNumber gem.